components/brave_wallet/browser/eth_pending_tx_tracker_unittest.cc

```diff
@@ -143,12 +140,14 @@ TEST_F(EthPendingTxTrackerUnitTest, UpdatePendingTransactions) {
       EthAddress::FromHex("0x2f015c60e0be116b1f0cd534704db9c92118fb6a");
   EthAddress addr2 =
       EthAddress::FromHex("0x2f015c60e0be116b1f0cd534704db9c92118fb6b");
-  EthJsonRpcController controller(brave_wallet::mojom::Network::Mainnet,
-                                  shared_url_loader_factory());
+  EthJsonRpcController controller(shared_url_loader_factory(), GetPrefs());
   EthTxStateManager tx_state_manager(GetPrefs(), controller.MakeRemote());
   EthNonceTracker nonce_tracker(&tx_state_manager, &controller);
   EthPendingTxTracker pending_tx_tracker(&tx_state_manager, &controller,
                                          &nonce_tracker);
+  base::RunLoop run_loop;
+  tx_state_manager.SetChainCallbackForTesting(run_loop.QuitClosure());
```

Comment:
There is no need for this, just use `base::RunLoop().RunUntilIdle()`

Reply:
done